### PR TITLE
Fix tailwind styles

### PR DESCRIPTION
**Pull request by josephjclark**
This is a suggested quick fix to the styling issues in Lightning.

* use the `prefix` option in Tailwind config to prefix all styles
* Sadly that means I've had to manually prefix every utility class used in the  components
* Remove base styles from the generated CSS (on the assumption that Lightning is using it)

This breaks the example a little bit but I don't think I mind at this stage as I see this as a temporary solution. If it ends up being permanent then, when I've stopped vomiting, I'll add tailwind to the example page and it should all work.

I've raised #84 to track the real issue, which I'll address as a priority this afternoon/next week.

Also, unrelated, apparently I didn't push my last commit to the main PR. Bit worrying. So I've included that, which updates the `export` utility script.

Oh, there's a little bit of tidying up of some unused stuff here as well.

### CODE REVIEW

File: packages/adaptor-docs/tailwind.config.cjs [hunk 1]
```diff
@@ -1,5 +1,6 @@
 // This is duplicated and lightly edited from Lightning
 module.exports = {
+  prefix: 'ad--',
```
Comment (josephjclark) [inline packages/adaptor-docs/tailwind.config.cjs:3]: There's quiet a noisy diff here (sorry!) but this is basically the fix

File: packages/adaptor-docs/src/DocsPanel.tsx [hunk 2]
```diff
@@ -23,9 +23,9 @@ const DocsPanel = ({ specifier, onInsert }: DocsPanelProps) => {
   
   const { name, version, functions } = pkg as PackageDescription;
   return (
-    <div className="block m-2">
-      <h1 className="h1 text-lg font-bold text-secondary-700 mb-2">{name} v{version}</h1>
-      <div className="text-sm mb-4">Operations available for use by this adaptor are listed below.</div>
+    <div className="ad--block ad--m-2">
```
Comment (josephjclark) [inline packages/adaptor-docs/src/DocsPanel.tsx:26]: This is also part of the actual fix - repeated across both components in the app.

**Comment by stuartc** [comment]
I'd prefer to try the approach of not bundling the css (via injection) for now - assuming we don't run into any issues with the approach here https://github.com/OpenFn/kit/pull/87.

If we want to really get into prefixing in order to have truly isolated styling then this is the way to go - and if we do I'd like to make sure we have a rock solid way to both minify and dedup.

Long story short, I'm starting to think components shouldn't be injecting styles - consumers can but not our deps? Your thoughts?

**Comment by josephjclark** [comment]
I will leave this open until next week because I still need to move the export utility across, so it'll remind me of that. Then I'll shut it down.

I do agree that the component shouldn't be injecting their own CSS. Especially tailwind components because they're all going to be injecting the same stuff.

I'll reserve more comment until I get a chance to do more research next week :)

**Pull request closed without merging**